mysql/gcpmysql/gcpmysql_test.go

```diff
@@ -130,3 +132,80 @@ func TestInstanceFromURL(t *testing.T) {
 		})
 	}
 }
+
+func Test_configWithOptions(t *testing.T) {
+	type args struct {
+		urlStr string
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    *drvr.Config
+		wantErr bool
+	}{
+		{
+			name: "no options",
+			args: args{
+				urlStr: "gcpmysql://my-project-id/us-central1/my-instance-id/my-db",
+			},
+			want:    &drvr.Config{},
+			wantErr: false,
+		},
+		{
+			name: "single option",
+			args: args{
+				urlStr: "gcpmysql://my-project-id/us-central1/my-instance-id/my-db?parseTime=true",
+			},
+			want: func() *drvr.Config {
+				cfg := drvr.NewConfig()
+				cfg.Net = "tcp"
+				cfg.Addr = "127.0.0.1:3306"
```

Comment:
For example, we won't see an `Addr` like this anymore, but rather the real one from the URL.